core/src/datayoga_core/blocks/relational/utils.py

```diff
@@ -31,6 +31,7 @@ def get_engine(connection_name: str, context: Context, autocommit: bool = True)
     db_type = DbType(connection.get("type", "").lower())
 
     extra = {}
+    ssl_args = {}
```

Comment:
maybe add comments that explain the extra and ssl_args.

Reply:
✅